dvc/output.py

```diff
@@ -1501,6 +1507,7 @@ def _merge_dir_version_meta(self, other: "Output"):
     **ANNOTATION_SCHEMA,
     Output.PARAM_CACHE: bool,
     Output.PARAM_REMOTE: str,
+    Output.PARAM_PULL: vol.All(vol.Lower, Choices("always", "never", "explicit")),
```

Comment:
I opted for `always` and `never` instead of `true` and `false` to avoid confusion with YAML types.

Reply:
I think it's fine to use YAML types and true / false. Also, I don't think we need the third option. I would make `false` behavior that same as explicit to keep things simpler.